### PR TITLE
fix(deploygroup_static): fix and harmonize massive action

### PR DESCRIPTION
**Pull request by stonebuzz**
Actually specific massives actions ```add to group``` or ```remove from group``` are not displayed

image

Now : 

From ```criteria``` tab we can add computer to group easily
image

From ```Associated elements``` tab we can delete computer to group easily
image

This PR try to harmonize massives actions from computer by adding ```exclude_from_group```  (```add_to_group``` already present```)

image

fix : https://github.com/glpi-project/glpi-inventory-plugin/issues/308

### CODE REVIEW

**Comment by trasher** [comment]
This is a part I absolutely do not know... I'm not fan of override of computer, but it seems it's been done that way in the original plugin?
Also, the new file license header is wrong

File: inc/computer.class.php [hunk 1]
```diff
@@ -0,0 +1,180 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ * GLPI Inventory Plugin
+ * Copyright (C) 2021 Teclib' and contributors.
```
Comment (trasher) [inline inc/computer.class.php:6]:
```suggestion
 * Copyright (C) 2021-2023 Teclib' and contributors.
```

File: inc/computer.class.php [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+ *
+ * http://glpi-project.org
+ *
+ * based on FusionInventory for GLPI
+ * Copyright (C) 2010-2021 by the FusionInventory Development Team.
```
Comment (trasher) [inline inc/computer.class.php:11]:
```suggestion
 * Copyright (C) 2010-2023 by the FusionInventory Development Team.
```

**Review by trasher — APPROVED** [review]
Seems correct, but I cannot test that

**Review by cedric-anne — APPROVED** [review]
Same for me.

**Pull request merged**